class-tgm-plugin-activation.php

```diff
@@ -726,28 +752,19 @@ public function notices() {
 			$activate_link       = false;   // Set to false, change to true in loop if conditions exist, used for action link 'activate'.
 			$activate_link_count = 0;       // Used to determine plurality of activate action link text.
 
-			foreach ( $this->plugins as $plugin ) {
+			foreach ( $this->plugins as $slug => $plugin ) {
```

Comment:
Should be done in a separate PR, is a different issue. (and I'm preparing a number of those already, mainly focussed on abstracting (nearly) duplicate code to functions)